test/test_amici_objective.py

```diff
@@ -63,7 +63,7 @@ def test_preeq_guesses(self):
             problem=problem, optimizer=optimizer, n_starts=2,
         )
 
-        self.assertTrue(obj.steadystate_guesses['fval'] < np.inf)
+        self.assertTrue(problem.objective.steadystate_guesses['fval'] < np.inf)
```

Comment:
@FFroehlich if you used this in your code --- note that the objective is deep-copied when put into the problem. thus the original object does not hold any information afterwards.

Reply:
Should be fine, not currently used :)

Reply:
Funnily, it was only filled due to a wrong copy action in the first place :see_no_evil: